### PR TITLE
Initial component setup

### PR DESCRIPTION
**Pull request by Peppe**

### CODE REVIEW

File: web-test-runner.config.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/* eslint-env node */
+const { createSauceLabsLauncher } = require('@web/test-runner-saucelabs');
+
+const config = {
+  nodeResolve: true,
+  browserStartTimeout: 60000, // default 30000
```
Comment (web-padawan) [inline web-test-runner.config.js:6]: Consider using `120000` for iOS as that seems to be more reliable.

File: web-test-runner.config.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    }),
+    sauceLabsLauncher({
+      browserName: 'safari',
+      platformName: 'macOS 10.15',
+      browserVersion: '13.1'
+    }),
+    sauceLabsLauncher({
+      browserName: 'iphone',
+      platform: 'iPhone X Simulator',
+      version: '13.0'
```
Comment (web-padawan) [inline web-test-runner.config.js:54]: Consider using `14.0`

File: src/vaadin-messages.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    super.ready();
+    this.setAttribute('value', 'hello world');
+  }
+
+  static get is() {
+    return 'vaadin-messages';
+  }
+
+  static get version() {
+    return '1.0.0-alpha1';
```
Comment (web-padawan) [inline src/vaadin-messages.js:96]: Please update this to `0.1.0` as the version needs to be bumped to `1.0.0-alpha1` during the first release.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@vaadin/vaadin-messages",
+  "version": "1.0.0-alpha1",
```
Comment (web-padawan) [inline package.json:3]: Please update this to `0.1.0` as the version needs to be bumped to `1.0.0-alpha1` during the first release.

File: test/messages.test.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    expect(value.textContent).to.be.equal('hello world');
+  });
+
+  it('should say hello tests', () => {
+    messages.value = 'hello tests';
+    expect(value.textContent).to.be.equal('hello tests');
+  });
+});
+
+describe('messages in column flex', () => {
```
Comment (web-padawan) [inline test/messages.test.js:27]: These tests and related styles can be probably removed.

File: web-test-runner.config.js (hunk 1 above)
Comment (Peppe) [reply web-test-runner.config.js]: Done.

File: web-test-runner.config.js (hunk 2 above)
Comment (Peppe) [reply web-test-runner.config.js]: Done.

File: src/vaadin-messages.js (hunk 3 above)
Comment (Peppe) [reply src/vaadin-messages.js]: Done.

File: package.json (hunk 4 above)
Comment (Peppe) [reply package.json]: Done.

File: test/messages.test.js (hunk 5 above)
Comment (Peppe) [reply test/messages.test.js]: Done.

**Review by web-padawan — APPROVED** [review]

**Pull request merged**